tests/conftest.py

```diff
@@ -155,17 +155,19 @@ def byo_cpu():
 
 @pytest.fixture(scope="session")
 def v100_gpu_cluster():
-    return rh.cluster("^rh-v100", provider="aws").up_if_not()
+    return rh.autocluster("^rh-v100", provider="aws").up_if_not()
```

Comment:
Won't work anymore, because we don't allow overwriting configs if a name is passed